apps/email/js/sync.js

```diff
@@ -77,22 +75,23 @@ define(function(require) {
           });
         }
 
-        title = title || mozL10n.get('notification-no-subject');
-
-        var notification = new Notification(title, notificationOptions);
-
-        // If the app is open, but in the background, when the notification
-        // comes in, then we do not get notifived via our mozSetMessageHandler
-        // that is set elsewhere. Instead need to listen to click event
-        // and synthesize an "event" ourselves.
-        notification.onclick = function() {
-          evt.emit('notification', {
-            clicked: true,
-            imageURL: iconUrl,
-            tag: notificationId,
-            data: data
-          });
-        };
+        titleL10n = { raw: titleL10n } || 'notification-no-subject';
```

Comment:
Instead of this you have to use:

```javacscript
titleL10n = titleL10n || 'notification-no-subject';
```

and then format `titleL10n` in `sendNotification` callers.